lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -472,8 +472,11 @@ sub stat_sensitivity_exact {
     my $results = $self->get_rawstats();
 
     my $total_true = $self->stat_total_true_positive_exact;
-    if ($total_true == 0) {
-        return 0;
+    {
+        no warnings;
```

Comment:
What warnings are you trying to avoid?  It's probably better to only disable the warnings you expect to see, e.g.:
```perl
no warnings qw(uninitialized);
```
although another way to avoid `undef` here could be:
```perl
my $total_true = $self->stat_total_true_positive_exact // 0;
```